src/Controller/ActivityController.php

```diff
@@ -1446,4 +1449,293 @@ public function declineInvitation(
         $activityUserRepo->save($accept);
         return new JsonResponse(['message' => 'You declined the invitation!'], Response::HTTP_OK);
     }
+
+    /**
+     * Add comment for activity.
+     * @Rest\Post("/{id}/add_comment", requirements={"id"="\d+"})
+     * @SWG\Post(
+     *     tags={"Activity"},
+     *     summary="Add comment.",
+     *     description="Add comment.",
+     *     operationId="addComment",
+     *     produces={"application/json"},
+     *     @SWG\Parameter(
+     *     description="ID of Activity",
+     *     in="path",
+     *     name="id",
+     *     required=true,
+     *     type="integer",
+     * ),
+     *     @SWG\Parameter(
+     *     description="Json body for the request",
+     *     name="requestBody",
+     *     required=true,
+     *     in="body",
+     *     @Model(type=CommentDTO::class, groups={"AddComment"}),
+     * )
+     * )
+     * @SWG\Response(
+     *     response=401,
+     *     description="Unauthorized.",
+     *     @SWG\Schema(
+     *     @SWG\Property(property="code", type="integer", example=401),
+     *     @SWG\Property(property="message", type="string", example="JWT Token not found"),
+     *     )
+     * )
+     * @SWG\Response(
+     *     response="200",
+     *     description="Successfull operation!",
+     *     @SWG\Schema(
+     *     @SWG\Property(property="message", type="string", example="Your comment has successfully added!"),
+     *     )
+     * )
+     * @SWG\Response(
```

Comment:
Response code 403 is never shown in a response. Maybe a validation is needed (you shouldn't be able to comment on a unavailable/private activity)

Reply:
Nice catch 👍